concourse/pipelines/deploy.yml

```diff
@@ -55,7 +55,7 @@ resources:
     icon: concourse-ci
     source:
       <<: *govuk-infrastructure-source
-      paths: [ concourse/tasks, lib/signon, tasks/secretsmanager.rake, tasks/app_secrets.rake ]
+      paths: [ concourse/tasks, lib/signon, lib/tasks/secretsmanager.rake, lib/tasks/app_secrets.rake ]
```

Comment:
Maybe put the whole of `lib/tasks` here? Will there be any tasks in there that shouldn't create new versions of this resource?

Reply:
There's `lib/tasks/{lint,test}.rake` too. Maybe I should put these tasks into a sub directory or create a `concourse` file.

Reply:
Cool, not that important. Approved 😄